sql/migrations/warehouse/000015_add_wh_async_jobs.up.sql

```diff
@@ -0,0 +1,22 @@
+--
+-- wh_staging_files
+--
+
+CREATE TABLE wh_async_jobs (
```

Comment:
It makes sense. But it will involve some significant changes. Let me see what I can do about this.

Reply:
I would prefer to have proper data types set up for the columns otherwise we would face issues in the long run around Query performance. But I don't think there would be major changes.

Reply:
Sure. Will make these changes.